app/controllers/kondos_controller.rb

```diff
@@ -1,4 +1,6 @@
 class KondosController < ApplicationController
+before_action :set_kondo, only: %i[show edit update destroy]
+
   def index
     @kondos = Kondo.order(created_at: :desc)
```

Comment:
Shante you didn't add this but we already removed this before but it came back lol. Do you mind just removing this line please.